hydra/controllers/client.py

```diff
@@ -816,3 +816,54 @@ def disable_metrics(self):
 
         self.restart_service()
         self.app.log.info(f'Successfully disabled all metrics reporting.')
+
+    @ex(
+        arguments=[
+            (
+                ['-n', '--name'],
+                {
+                    'help': 'name of network for key backup',
+                    'action': 'store',
+                    'dest': 'name'
+                }
+            ),
+            (
+                ['-D', '--destroy'],
+                {
+                    'help': 'Use to overwrite existing backups',
+                    'action': 'store_true',
+                    'dest': 'destroy'
+                }
+            ),
+        ]
+    )
+    def backup_nodekeys(self, destroy=False):
+        name = self.app.utils.env_or_arg(
+            'name', 'HYDRA_NETWORK', or_path='.hydra_network', required=True)
+        filenames = ['node_key.json',
+                     'priv_validator.json']
```

Comment:
Good idea, done.

Reply:
I ended up adding params for source and destination in case user wants to specify a directory different from the default ~/.hydra and /chaindata/config directories.